lib/Parse/Lexer.cpp

```diff
@@ -1316,6 +1340,62 @@ static const char *skipToEndOfInterpolatedExpression(const char *CurPtr,
   }
 }
 
+/// extract content of string literal from inside quotes
+static StringRef getStringLiteralContent(const Token &Str) {
+  StringRef Bytes = Str.getText();
+  Bytes = Bytes.drop_front().drop_back();
+
+  if (Str.getStringModifiers() & StringLiteralMultiline) {
+    Bytes = Bytes.drop_front().drop_back();
+    Bytes = Bytes.drop_front().drop_back();
+  }
+
+  return Bytes;
+}
+
+/// determine contents of literal to be normalised - either
+/// to strip indenting or normalise line endings to a single \n
```

Comment:
Nit: while you're capitalizing and re-wrapping these comments to 80 characters, U.S. English spellings. (Yes, I know; I'm Canadian, but them's the rules...)